src/components/button-toggle/button-toggle.html

```diff
@@ -0,0 +1,13 @@
+<label [attr.for]="inputId" class="md-button-toggle-label">
+  <input #input class="md-button-toggle-input"
+         [type]="type"
+         [id]="inputId"
+         [checked]="checked"
+         [disabled]="disabled"
+         [name]="name"
+         (change)="onClick()">
```

Comment:
It was causing odd issues with the radio vs the checkbox but I just fixed that in another branch, I'll move it over to this PR though.